site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst

```diff
@@ -798,6 +799,16 @@ for defining the binding:
    of one tool over the other will usually be based on which is the most
    natural fit for the project and its build system.
 
+.. _interacting-with-code-emnapi:
+
+Binding C/C++ and JavaScript — Emnapi (Node-API for Emscripten)
```

Comment:
Is that `-` some kind of unicode `-` ?  It looks different to the other ones in this file?  Can we stick the to ascii `-`?

Reply:
It was copied from this line

```
Binding C++ and JavaScript — WebIDL Binder and Embind
```